account-keys/src/account_keys.rs

```diff
@@ -24,6 +24,10 @@ use core::{
 };
 use curve25519_dalek::{ristretto::RistrettoPoint, scalar::Scalar};
 use mc_account_keys_types::RingCtAddress;
+use mc_core::{
+    prelude::{RootSpendPrivate, RootViewPrivate},
+    slip10::Slip10Key,
+};
 use mc_crypto_digestible::Digestible;
 use mc_crypto_hashes::{Blake2b512, Digest};
 use mc_crypto_keys::{RistrettoPrivate, RistrettoPublic};
```

Comment:
Do you plan to eventually replace these `RistrettoPublic` types with `RootViewPublic` and `RootSpendPublic`? (Curious if that's in the works and you just didn't want to do it in this PR because it'd make it too large.)

Reply:
sorta! the idea is that eventually all instances of `RistrettoX` would be replaced by the types in `mc_core_types::keys` to reduce opportunities for misuse (like the classic swapping spend and view keys misadventure). 

i need compatible types for the hardware wallet work, and will propagate these in the areas i'm working on (primarily to remove dependencies on `mc-account-keys`), but i'm hoping that once we have the types available other folks will pick up and run with this in other areas of the codebase.

Reply:
Got it, thanks!